packages/amplify-category-storage/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aws-amplify/amplify-category-storage",
-  "version": "4.0.0-cdkv2.2",
+  "version": "4.0.1-beta.0",
```

Comment:
is storage the only category that is in beta?

Reply:
yeah, there is this remnant on npm
![image](https://user-images.githubusercontent.com/5849952/202523631-755ce473-fae8-4e4e-b957-971d085a3bea.png)

Reply:
it will fail again when releasing next patch version

Reply:
No, we went to `4.0.1-beta.1` to avoid that.

Reply:
https://www.npmjs.com/package/@aws-amplify/amplify-category-storage/v/4.0.1-beta.1